macOS/DuckDuckGo/Tab/ViewModel/TabViewModel.swift

```diff
@@ -652,6 +652,6 @@ private extension NSAttributedString {
     static let releaseNotesTrustedIndicator = trustedIndicatorAttributedString(with: .releaseNotesIndicator,
                                                                                title: UserText.releaseNotesTitle)
     static let aiChatTrustedIndicator = trustedIndicatorAttributedString(with: .aiChatPreferences,
-                                                                         title: UserText.aiFeatures)
+                                                                         title: "Duck.ai")
```

Comment:
@ayoy I didn’t add this to `UserText`, it doesn’t make much sense to translate this, given that it is a product and it will be `Duck.ai` in all languages.